src/Microsoft.DotNet.Cli.Utils/FileAccessRetryer.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.DotNet.Cli.Utils
 {
     public static class FileAccessRetrier
     {
-        public static async Task<T> RetryOnFileAccessFailure<T>(Func<T> func, int maxRetries = 3000, TimeSpan sleepDuration = default(TimeSpan)) 
+        public static async Task<T> RetryOnFileAccessAssetsFileFailure<T>(Func<T> func, int maxRetries = 3000, TimeSpan sleepDuration = default(TimeSpan)) 
```

Comment:
Why the rename of this method?

Reply:
the name is misleading. I try to reuse it in the beginning and realize the error it throws is specific about Assets files

Reply:
plus async and `Func<T>` it is easy to create a new one

Reply:
Ugh, given how this is intended to be a generic utility class, I think it would be better served by keeping the generic name and not making the error message about the assets file.